components/css/componentPage.scss

```diff
@@ -69,3 +70,15 @@
   text-align: left;
   display: inline-block;
 }
+
+:global(.spectrum-CSSExample-example--overlay) {
+  background: rgba(0,0,0,0.4);
```

Comment:
no Spectrum var here for black?

Reply:
@bernhard-adobe there should be, but not with the alpha I don't think there is one with the correct the alpha.

Reply:
ah ok, thanks for the clarification.